images/metering-ansible-operator/roles/meteringconfig/defaults/main.yml

```diff
@@ -31,41 +31,53 @@ meteringconfig_hadoop_default_image_tag: "{{ _hadoop_default_spec.image.tag }}"
 meteringconfig_ghostunnel_default_image_repo: "{{ _ghostunnel_default.image.repository }}"
 meteringconfig_ghostunnel_default_image_tag: "{{ _ghostunnel_default.image.tag }}"
 
+meteringconfig_reporting_operator_override_image_repo: "{{ lookup('env', 'METERING_REPORTING_OPERATOR_IMAGE') }}"
+meteringconfig_oauth_proxy_override_image_repo: "{{ lookup('env', 'OAUTH_PROXY_IMAGE') }}"
+meteringconfig_presto_override_image_repo: "{{ lookup('env', 'METERING_PRESTO_IMAGE') }}"
+meteringconfig_hive_override_image_repo: "{{ lookup('env', 'METERING_HIVE_IMAGE') }}"
+meteringconfig_hadoop_override_image_repo: "{{ lookup('env', 'METERING_HADOOP_IMAGE') }}"
+meteringconfig_ghostunnel_override_image_repo: "{{ lookup('env', 'GHOSTUNNEL_IMAGE') }}"
+
 _storage_spec: "{{ meteringconfig_default_values | combine(meteringconfig_spec_overrides) | json_query('storage') | default({}, true) }}"
 
 # Override the default images we use with env vars if set, falling back to the values.yaml configured default if not set.
 meteringconfig_default_image_overrides:
   reporting-operator:
     spec:
       image:
-        repository: "{{ lookup('env','METERING_REPORTING_OPERATOR_IMAGE').split(':')[0] | default(meteringconfig_reporting_operator_default_image_repo, true) }}"
-        tag: "{{ lookup('env','METERING_REPORTING_OPERATOR_IMAGE').split(':')[1] | default(meteringconfig_reporting_operator_default_image_tag, true) }}"
+        repository: "{{ meteringconfig_reporting_operator_default_image_repo }}"
+        tag: "{{ meteringconfig_reporting_operator_default_image_tag }}"
+        override: "{{ meteringconfig_reporting_operator_override_image }}"
```

Comment:
where are any of these override images defined? I only see the override repository variables being defined above.

Reply:
https://storage.googleapis.com/origin-ci-test/pr-logs/pull/operator-framework_operator-metering/932/pull-ci-operator-framework-operator-metering-master-metering-e2e-aws/1207/artifacts/metering-e2e-aws/pods/metering-integration_metering-operator-799c659bb-p6gp5_ansible.log

Reply:
Just a mismatch in naming.